.github/workflows/deployment-template.yml

```diff
@@ -0,0 +1,55 @@
+name: OpenSearch and Dashboards Deployment Template
```

Comment:
could we add readme for .github/workflows. explains the usage of each file and relationship